src/components/modal/styles.scss

```diff
@@ -1,4 +1,29 @@
-.c-modal {
+@import 'styles/settings';
 
+ .c-modal {
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
+  background-color: rgba(255, 255, 255, 0.75);
+
+  .modal-content {
+    position: absolute;
+    top: 40px;
+    left: 40px;
+    right: 40px;
+    bottom: 40px;
+    border: 1px solid rgb(204, 204, 204);
```

Comment:
Try to avoid rgb colors... Only rgba should be used. 
Reason: hex colors will be easier to read